app/containers/Applications/ApplicationDetailsContainer.tsx

```diff
@@ -156,7 +170,72 @@ export const ApplicationDetailsComponent: React.FunctionComponent<Props> = ({
           />
         ))}
       </div>
+      <Card
+        style={{ width: "100%", marginBottom: "10px" }}
+        className="verification-statement-summary-card"
+      >
+        <Card.Header className="summary-form-header">
+          <Row>
+            <Col md={6}>
+              <h2>Verification Statement </h2>
+            </Col>
+            <Col md={1} style={{ textAlign: "right" }}>
+              <Button
+                aria-label="toggle-card-open"
+                title="expand or collapse the card"
+                variant="outline-dark"
+                onClick={() => setIsOpen(!isOpen)}
+              >
+                {isOpen ? "+" : "-"}
+              </Button>
+            </Col>
+          </Row>
+        </Card.Header>
+        <Collapse in={!isOpen}>
+          <Card.Body className="card-body">
+            {attachments ? (
+              attachments.map((opt) => (
```

Comment:
could use `{node}` as the map argument and then you don't have to repeat `opt.` everywhere